api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -25,102 +27,109 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 const (
-	ConfigCRName                         = "self-node-remediation-config"
-	defaultWatchdogPath                  = "/dev/watchdog"
-	DefaultSafeToAssumeNodeRebootTimeout = 180
-	defaultIsSoftwareRebootEnabled       = true
+	ConfigCRName        = "self-node-remediation-config"
+	defaultWatchdogPath = "/dev/watchdog"
+	// FallbackRebootDuration is used by the manager in case it fails to calculate the minimum safe reboot time
+	FallbackRebootDuration         = 300 * time.Second
```

Comment:
Even the existence of this const scares me, but what on what basis do we have confidence that 5min is sufficient?
Seems like that should at least be documented here.

If we agree this const should exist, the agent/system should arguably consider it an error if they or the watchdog are configured with values that would result in the agent taking longer.  I don't see any checks for this.

Reply:
>  I don't see any checks for this.

there can't be any checks because this was the fallback in case we run into issues when calculating the minimum reboot duration. However, this fallback is removed now, we retry now (see https://github.com/medik8s/self-node-remediation/pull/214#discussion_r1643184005)